StepDefinitions/AddlangStepDefinitions.cs

```diff
@@ -0,0 +1,124 @@
+using OpenQA.Selenium;
+using System;
+using SpecProj2.Pages;
+using TechTalk.SpecFlow;
+using System.Collections.Generic;
+using TechTalk.SpecFlow;
+using OpenQA.Selenium.Chrome;
+using NUnit.Framework;
+using System.Reflection.Emit;
+using OpenQA.Selenium.Support.UI;
+
+namespace SpecProj2.StepDefinitions
+{
+    [Binding]
+    public class AddlangStepDefinitions
+    {
+        IWebDriver driver = new ChromeDriver();
+        Login Loginobj = new Login();
+        Addmultiplelanguage Addmultiplelanguageobj = new Addmultiplelanguage();
+        Editlanguage Editlanguageobj = new Editlanguage();
+        Deletelanguage deletelanguageobj = new Deletelanguage();
+
+        [Given(@"User logs into Mars Portal")]
+        public void GivenUserLogsIntoMarsPortal()
+        {
+            Loginobj.login(driver);
+        }
+        [When(@"User creates a new Language record '([^']*)'")]
+        public void WhenUserCreatesANewLanguageRecord(string language)
+        {
+            Addmultiplelanguageobj.cleardata(driver);
```

Comment:
so Addmultiplelanguageobj.cleardata(driver); needs to be put in a hook ( one that runs before any test), and then at the end of this test you should only clean up the data this tests adds